osquery/remote/enroll/enroll.cpp

```diff
@@ -33,7 +33,7 @@ CLI_FLAG(bool,
          "Disable enrollment functions on related config/logger plugins");
 
 /// Path to optional enrollment secret data, sent with enrollment requests.
-CLI_FLAG(string,
+FLAG(string,
```

Comment:
I'm leery of changing this

Reply:
Yes, please do not change this, it is required for configuration, chicken and egg problem so to speak.